libraries/triumph-cmds/flags.md

```diff
@@ -8,15 +8,31 @@
 # Flags
 Flags, if you didn't already know, when in commands are typically short identifiers that convey bits of information
 or mark what certain arguments are going to be explicitly. It's useful for controlling lots of optional arguments.
-In commands, the argument would be the `-n` part of `/example flag -n=3`.
+In commands, the argument would be the `-n` part of `/flag example -n=1`.
 
 ## Example of Registering Flags
+
+### Without Flag Argument
+```java
+@Command("flag")
+public class Command {
+    
+    @SubCommand("example")
+    @CommandFlags({@Flag(flag = "f")})
+    public void execute(Sender sender, Flags flags) {
+        flags.getValue("f").ifPresent(arg -> {}); // no argument value since no argument
+    }
+    
+}
+```
+
+### With Flag Argument
 ```java
-@Command("example")
+@Command("flag")
 public class Command {
     
-    @SubCommand("flag")
-    @CommandFlags({@Flag(flag = "n", longFlag = "num", argument = int.class, suggestion = "[<number>]")})
+    @SubCommand("example")
+    @CommandFlags({@Flag(flag = "n", longFlag = "num", argument = int.class, suggestion = "1")})
```

Comment:
The suggestion is for suggestions keys, so use for example `"numbers"`

Reply:
Ahhhhhhhhhh.... ok ok makes more sense